src/test/java/com/algorand/algosdk/transaction/TestTransaction.java

```diff
@@ -503,7 +502,7 @@ public void testMakeAssetAcceptanceTxn() throws Exception {
         SignedTransaction stxDecoded = Encoder.decodeFromMsgPack(encodedOutBytes, SignedTransaction.class);
```

Comment:
Is this check here and elsewhere now redundant?
Isn't this already checked in serializeDeserializeCheck?

Reply:
I think so, but assuming the golden is shared across SDKs I think it's still a good safeguard.